tests/bulk/README.md

```diff
@@ -163,6 +163,19 @@ def get_or_init_instance(self, instance_loader, row):
 | `use_bulk=True, batch_size=None`   | 14.08       | 39.40         |
 | `use_bulk=True, batch_size=1000`   | 15.37       | 32.70         |
 
+#### bulk export
+
+Should show a clear difference in performance depending on the number of fields selected.
+
+```bash
+./manage.py runscript bulk_export
+```
+
+| Condition      | Time (secs) | Memory (MB) |
+| -------------- | ----------- | ----------- |
+| limited export | 0.2999      | 0.0273      |
+| normal export  | 0.0625      | 0.0625      |
```

Comment:
@bluppfisk does this mean that it would be better not to introduce this PR? I see the memory usage goes down but time goes up...

Reply:
I had things the wrong way around and I've edited my comment since. Sorry :)

Feel free to repeat the test yourselves with both codebases.

Reply:
btw - this `README.md` file will be removed in v3.  So this addition is not necessary.  The comments in this PR can serve as a record of the performance improvement.